### PR TITLE
New version: Shadow.Shadow version 8.0.10018

### PR DESCRIPTION
**Pull request by mdanish-kh**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----
- Closes #87833

Modified the `PackageIdentifier` from `Blade.Shadow` -> `Shadow.Shadow` as the Publisher field has been updated in Control Panel.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/87834)

### CODE REVIEW

**Review by ItzLevvie — DISMISSED** [review]

**Comment by mdanish-kh** [comment]
Marking PR as draft as the PackageIdentifier may need changing here. Will rebase the PR once I get the approval.

**Comment by mdanish-kh** [comment]
@Trenly - Could you please confirm https://github.com/microsoft/winget-pkgs/pull/87833#issuecomment-1307891638 so that I can rebase this PR accordingly.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**
